MSAL/src/MSALPublicClientApplication.m

```diff
@@ -116,16 +134,34 @@ - (id)initWithClientId:(NSString *)clientId
 
 - (NSArray <MSALUser *> *)users:(NSError * __autoreleasing *)error
 {
-    return [_tokenCache getUsers:self.clientId context:nil error:error];
+    NSMutableSet *users = [NSMutableSet new];
```

Comment:
just curious, why not use array?

Reply:
If we use array here, we will return 2 same users. It happens because we store same RT in 2 formats.